test/integration/user_can_add_stache_to_cart_from_show_page_test.rb

```diff
@@ -0,0 +1,14 @@
+require 'test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.